File: Word-59A.py
# https://codeforces.com/problemset/problem/59/A
def correct_word(word):
    uppercase_count = sum(1 for char in word if char.isupper())
    lowercase_count = len(word) - uppercase_count

    if uppercase_count > lowercase_count:
        corrected_word = word.upper()
    else:
        corrected_word = word.lower()

    return corrected_word

word = input().strip()

result = correct_word(word)
print(result)

File: Word-59A.c
// https://codeforces.com/problemset/problem/59/A
#include <stdio.h>
#include <ctype.h>
#include <string.h>

void correct_word(char* word) {
    int uppercase_count = 0;
    int lowercase_count = 0;
    int length = strlen(word);

    for (int i = 0; i < length; i++) {
        if (isupper(word[i])) {
            uppercase_count++;
        } else {
            lowercase_count++;
        }
    }

    if (uppercase_count > lowercase_count) {
        for (int i = 0; i < length; i++) {
            word[i] = toupper(word[i]);
        }
    } else {
        for (int i = 0; i < length; i++) {
            word[i] = tolower(word[i]);
        }
    }
}

int main() {
    char word[101];  // Assuming the word won't exceed 100 characters
    scanf("%100s", word);

    correct_word(word);

    printf("%s\n", word);

    return 0;
}
